meshnesh/react-consume-api
# Ft auth user logout && tests

# What does this PR do?

This PR allows a user to logout of the application.

## How should this be manually tested?

* Clone the repo.
* Install the packages npm install then run npm run dev to run the app.
* Create an account on the app by registering, confirm your email address.
* Click on logout to logout of the app.
